content/post/deploy-mssql-server-in-aks/index.md

```diff
@@ -0,0 +1,454 @@
+
+NAME                                       TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+service/mssqlserver-ag-cluster             ClusterIP   10.128.113.192   <none>        1433/TCP   7m41s
+service/mssqlserver-ag-cluster-pods        ClusterIP   None             <none>        1433/TCP   7m41s
+service/mssqlserver-ag-cluster-secondary   ClusterIP   10.128.213.243   <none>        1433/TCP   7m41s
+
+NAME                                                        TYPE                     VERSION   AGE
+appbinding.appcatalog.appscode.com/mssqlserver-ag-cluster   kubedb.com/mssqlserver   2022      6m37s
+
+NAME                                            VERSION     STATUS   AGE
+mssqlserver.kubedb.com/mssqlserver-ag-cluster   2022-cu12   Ready    7m41s
+```
+Let’s check if the database is ready to use,
+
+```bash
+$ kubectl get mssqlserver -n demo mssqlserver-ag-cluster
+NAME                     VERSION     STATUS   AGE
+mssqlserver-ag-cluster   2022-cu12   Ready    8m16s
+```
+> We have successfully deployed MSSQL Server in AKS. Now we can exec into the container to use the database.
+
+
+### Accessing Database Through CLI
+
+To access your database through the CLI, you first need the credentials for the database. KubeDB will create several Kubernetes Secrets and Services for your MSSQL Server instance. To view them, use the following commands:
+
+```bash
+$ kubectl get secret -n demo -l=app.kubernetes.io/instance=mssqlserver-ag-cluster 
+NAME                                   TYPE                       DATA   AGE
+mssqlserver-ag-cluster-auth            kubernetes.io/basic-auth   2      9m36s
+mssqlserver-ag-cluster-client-cert     kubernetes.io/tls          3      9m36s
+mssqlserver-ag-cluster-dbm-login       kubernetes.io/basic-auth   1      9m36s
+mssqlserver-ag-cluster-endpoint-cert   kubernetes.io/tls          3      9m36s
+mssqlserver-ag-cluster-master-key      kubernetes.io/basic-auth   1      9m36s
+mssqlserver-ag-cluster-server-cert     kubernetes.io/tls          3      9m36s
+
+$ kubectl get service -n demo -l=app.kubernetes.io/instance=mssqlserver-ag-cluster 
+NAME                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+mssqlserver-ag-cluster             ClusterIP   10.128.113.192   <none>        1433/TCP   9m43s
+mssqlserver-ag-cluster-pods        ClusterIP   None             <none>        1433/TCP   9m43s
+mssqlserver-ag-cluster-secondary   ClusterIP   10.128.213.243   <none>        1433/TCP   9m43s
+```
+
+From the above list, the `mssqlserver-ag-cluster-auth` Secret contains the admin-level credentials needed to connect to the database. Use the following commands to obtain the username and password:
+
+```bash
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.username}' | base64 -d
+sa
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.password}' | base64 -d
+pT42K58uKEe3pote
+```
+
+
+### Insert Sample Data
```

Comment:
Before inserting data, We need to identify the primary node first. Since we can write only in the primary node. `mssqlserver-ag-cluster-0` may not be the primary 

` To get the role, we can run this command` 
➤ kubectl get pods -n demo --show-labels
NAME           READY   STATUS    RESTARTS   AGE   LABELS
ms-ag-mon-0    3/3     Running   0          19h   app.kubernetes.io/component=database,app.kubernetes.io/instance=ms-ag-mon,app.kubernetes.io/managed-by=kubedb.com,app.kubernetes.io/name=mssqlservers.kubedb.com,apps.kubernetes.io/pod-index=0,controller-revision-hash=ms-ag-mon-d6b4f4c84,`kubedb.com/role=primary`,statefulset.kubernetes.io/pod-name=ms-ag-mon-0
ms-ag-mon-1    3/3     Running   0          19h   app.kubernetes.io/component=database,app.kubernetes.io/instance=ms-ag-mon,app.kubernetes.io/managed-by=kubedb.com,app.kubernetes.io/name=mssqlservers.kubedb.com,apps.kubernetes.io/pod-index=1,controller-revision-hash=ms-ag-mon-d6b4f4c84,`kubedb.com/role=secondary`,statefulset.kubernetes.io/pod-name=ms-ag-mon-1
ms-ag-mon-2    3/3     Running   0          19h   app.kubernetes.io/component=database,app.kubernetes.io/instance=ms-ag-mon,app.kubernetes.io/managed-by=kubedb.com,app.kubernetes.io/name=mssqlservers.kubedb.com,apps.kubernetes.io/pod-index=2,controller-revision-hash=ms-ag-mon-d6b4f4c84,`kubedb.com/role=secondary`,statefulset.kubernetes.io/pod-name=ms-ag-mon-2